### PR TITLE
Update release-email.md

### PR DESCRIPTION
**Pull request by volodymyrss**
started gw, will send to antares and gwosc

would be best to have approval of GW
would be good to have renku UI change, or at least https://github.com/oda-hub/dispatcher-app/issues/392

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
 ## User experience improvements
 
-* Some improvements in error reporting in frontend an API.
-* More clear note in the email about expiration time of the URL in the cases when these URLs are temporary.
-* When the URLs provided in job email and in the frontend are persistent they are more consistent: parameters in the URL are converted to default formats before being inserted to the URL (e.g. time will be always represented as ISOT).
+* Added [Gravitatitional Wave analysis](https://www.astro.unige.ch/mmoda/help/mmoda/gravitational-wave-analysis)
+* Added [DESI LegacySurvey analysis](https://www.astro.unige.ch/mmoda/help/mmoda/legacy-survey)
+* Added possibility to open a new [RenkuLab](https://renkulab.io/) session
+
+* Some improvements in error reporting in frontend an API
```
Comment (burnout87) [inline README.md]:
```suggestion
* Some improvements in error reporting in frontend and API
```

**Review by andriineronov — APPROVED** [review]

**Comment by volodymyrss** [comment]
while everybody is still there, we can send it tomorrow? or today, since tomorrow is a holiday in France? @andriineronov @ferrigno  ?

**Comment by andriineronov** [comment]
May be it's too late today for France (they would not ruh to read and try at the end-of-the day before July 14). May be next Monday? For GW I would add some neutral intro : they would see it for the first time.....

File: README.md [hunk 2]
```diff
@@ -1,20 +1,22 @@
...
+Send to SDSC, ANTARES, GWOSC, SDSS/LegacySurvery (if possible) INTEGRAL collaborations and all users
 
-`21.10.0003` is a patch release. For details on the current feature relese please see release notes of [21.10.0000](https://github.com/oda-hub/release-notes/tree/21.10.0000).
+----
 
-We fixed several bugs and improved user experience, as described below.
-Some of the changes simplify process of running large requests (over 20k ScW). 
-Note that for now, requests of this size can be only submitted upon special agreement.
+# Release notes 22.07.0000
+
```
Comment (volodymyrss) [inline README.md:5]: add neutral info here, @volodymyrss

**Comment by volodymyrss** [comment]
> May be it's too late today for France (they would not ruh to read and try at the end-of-the day before July 14). May be next Monday? 
> For GW I would add some neutral intro : they would see it for the first time.....

also DESI. Though this could go into only their personalized emails, we can add a generic introduction in the release notes.

**Comment by volodymyrss** [comment]
> May be it's too late today for France (they would not ruh to read and try at the end-of-the day before July 14). May be next Monday? For GW I would add some neutral intro : they would see it for the first time.....

will @ferrigno manage to send it on Monday? next week @ferrigno  is in the mountains!

File: README.md [hunk 3]
```diff
@@ -1,20 +1,64 @@
...
+* Added [Gravitatitional Wave analysis](https://www.astro.unige.ch/mmoda/help/mmoda/gravitational-wave-analysis)
+* Added [DESI LegacySurvey analysis](https://www.astro.unige.ch/mmoda/help/mmoda/legacy-survey)
+* Added possibility to open a new [RenkuLab](https://renkulab.io/) session
 
 ## User experience improvements
 
-* Some improvements in error reporting in frontend an API.
-* More clear note in the email about expiration time of the URL in the cases when these URLs are temporary.
-* When the URLs provided in job email and in the frontend are persistent they are more consistent: parameters in the URL are converted to default formats before being inserted to the URL (e.g. time will be always represented as ISOT).
+* Some improvements in error reporting in frontend and API
```
Comment (burnout87) [inline README.md:60]: I would mention also the new error reports within emails

Comment (volodymyrss) [reply README.md:60]: they are not added yet to the release, since they are not verified enough in live staging. If we verify we can add. For now - no.

**Review by burnout87 — APPROVED** [review]

**Comment by ferrigno** [comment]
I would really expand the message especially on new features: as it is, it is not enough communicative. Nobody knows renkulab, we should explain in a few sentences what it is. The same for gravitational waves, what is available and what is not. As of now, the message is not very catching.

**Comment by volodymyrss** [comment]
> I would really expand the message especially on new features: as it is, it is not enough communicative. Nobody knows renkulab, we should explain in a few sentences what it is. The same for gravitational waves, what is available and what is not. As of now, the message is not very catching.

We slightly expanded. Note that the links already point to the help pages with extra details. Please have a look and let us know when you can send it!

**Comment by ferrigno** [comment]
I made some edits, you can check if you like them. How can we convert Markdown in html to send the email ?

File: README.md [hunk 4]
```diff
@@ -7,8 +7,15 @@ Send to SDSC, ANTARES, GWOSC, SDSS/LegacySurvery (if possible) INTEGRAL collabor
...
-The underlying services run complex pipelines in a computer cluster based at the University of Geneva and the [web-based interface](http://astro.unige.ch/mmoda/) and a [python API](https://oda-api.readthedocs.io/en/latest/user_guide/tutorial_main.html) make them available in a straightforward way. Pipelines have been developed by experts in the data analysis and exploit proximity to the data combined with state-of-the-art calibration and software.
+The Multi-Messenger Online analysis is an [open-source](https://github.com/oda-hub) **cloud-native platform** to derive **high-level scientific products for several astrophysical instruments**.
+A [web-based interface](http://astro.unige.ch/mmoda/) and a [python API](https://oda-api.readthedocs.io/en/latest/user_guide/tutorial_main.html) allow the users to run easily the underlying services, which run complex pipelines in a computer cluster based at the University of Geneva. These pipelines have been developed by experts in the data analysis and exploit proximity to the data combined with state-of-the-art calibration and software.
+
+Currently, MMODA provides data from:
```
Comment (volodymyrss) [inline README.md:13]: this is rather duplicating what's at the end. But ok.

File: README.md [hunk 5]
```diff
@@ -46,7 +53,7 @@ MMODA is developed joinly by [UNIGE Department of Astronomy](https://www.unige.c
 
 * Added [Gravitatitional Wave analysis](https://www.astro.unige.ch/mmoda/help/mmoda/gravitational-wave-analysis) based on [GWOSC](https://www.gw-openscience.org/) analysis threads.
 * Added visible and infrared photometry from [DESI LegacySurvey](https://www.astro.unige.ch/mmoda/help/mmoda/legacy-survey) based on [NOIRLab](https://datalab.noirlab.edu/ls/dataAccess.php) TAP service.
-* Added possibility to open the MMODA results in a JupyterLab analysis environment at [RenkuLab](https://renkulab.io/).
+* Added possibility to open the MMODA results in a JupyterLab analysis environment at [RenkuLab](https://renkulab.io/). This allows the user to experience the potential of using a Jupyter notebook in a pre-built environment.
```
Comment (volodymyrss) [inline README.md]: the seconds sentence largely duplicates the first one. Also we wanted to keep the bullet points roughly same weight. I would remove the second sentence.

Comment (ferrigno) [reply README.md]: technically, you are right, but I am not sure most people know that the second sentence rephrases the first one. I will not make a big point out of this, though.

Comment (volodymyrss) [reply README.md]:
```suggestion
* Added possibility to open the MMODA results in a JupyterLab analysis environment at [RenkuLab](https://renkulab.io/). 
```

**Pull request merged**